### ISSUE TITLE
On the main menu of UNA, screen reader doesn't indicate buttons/images that can be selected.

### ISSUE THREAD

**Issue by Wajeed-msft** [MicrosoftDocs/msteams-docs#4732]
Copied bug from: https://github.com/microsoft/AdaptiveCards/issues/6777 

**Details from original post:**
### Target Application

Microsoft Teams

### Application Operating System

Windows

### Schema Version

v1.2

### Problem Description

On the main menu of UNA, screen reader reads that out the menu items but it doesn't indicate they are buttons/images that can be selected. In addition, after they are read, user cannot navigate or select any of the button/image afterwards.

### Expected Outcome

screen reader should indicate they are buttons/images that can be selected. In addition, after they are read, user should navigate or select any of the button/image afterwards.

### Actual Outcome

screen reader doesn't indicate they are buttons/images that can be selected. In addition, after they are read, user cannot navigate or select any of the button/image afterwards.

### Card JSON

```JSON
{
  "type": "AdaptiveCard",
  "body": [
    {
      "type": "ColumnSet",
      "columns": [
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "Navigation and Help",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Navigation and help",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with Navigation and Help"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "height": "stretch",
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": " Pay and benefits",
              "horizontalAlignment": "Center",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Pay and Benefits",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with Pay and benefits"
                  }
                }
              },
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": " Learning and development",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Learning and development",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with Learning and development"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        }
      ],
      "style": "default",
      "horizontalAlignment": "Center",
      "spacing": "Padding",
      "height": "stretch"
    },
    {
      "type": "ColumnSet",
      "columns": [
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": " joining moving and leaving",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Joining Moving and leaving",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with joining moving and leaving"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "tools",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Tools",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with the tools you have"
                  }
                }
              },
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "my workplace",
              "horizontalAlignment": "Center",
              "selectAction": {
                "type": "Action.Submit",
                "title": "My Workplace",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with my workplace"
                  }
                }
              },
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        }
      ],
      "style": "default",
      "horizontalAlignment": "Center",
      "spacing": "Padding",
      "height": "stretch"
    },
    {
      "type": "ColumnSet",
      "columns": [
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "finance",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Finance",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with Finance"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "My Una",
              "selectAction": {
                "type": "Action.Submit",
                "title": "My Una",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with My Una?"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        },
        {
          "type": "Column",
          "width": "auto",
          "items": [
            {
              "type": "Image",
              "url": "sample url",
              "altText": "My Travel",
              "selectAction": {
                "type": "Action.Submit",
                "title": "Learning and development",
                "data": {
                  "msteams": {
                    "type": "imBack",
                    "value": "Can you help me with Travel"
                  }
                }
              },
              "horizontalAlignment": "Center",
              "width": "100px"
            }
          ],
          "horizontalAlignment": "Center",
          "spacing": "Padding"
        }
      ],
      "style": "default",
      "horizontalAlignment": "Center",
      "spacing": "Padding",
      "height": "stretch"
    }
  ],
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "version": "1.2"
}
```

### Repro Steps

_No response_---
#### Document Details

⚠ *Do not edit this section.It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3fb4394d-25b1-8752-3996-fe3a2934732d
* Version Independent ID: 6d391381-89d0-117c-27cc-00043bd26a36
* Content: [Cards reference - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference)
* Content Source: [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/task-modules-and-cards/cards/cards-reference.md)
* Product: **msteams**
* GitHub Login: @surbhigupta12
* Microsoft Alias: **surbhigupta12**

**Comment by Prasad-MSFT** [comment]
@prajwalmanju - We are checking the issue. Will get back to you shortly

**Comment by Prasad-MSFT** [comment]
@prajwalmanju - Could you please provide more details or repro steps? What tool you are using here ?

**Comment by Prasad-MSFT** [comment]
@prajwalmanju - We tried to repro the issue with Narrator and can confirm that it doesn't indicate by default if the button/image can be selected but if you hover on that particular button/image it explains it can be navigable. Please let us know if you are using the same tool or not.

**Comment by prajwalmanju** [comment]
Hello Team,

We are using NVDA and JAWS tool to reproduce this issue.

Thanks!

**Comment by Prasad-MSFT** [comment]
@prajwalmanju -We tried to repro the issue with NVDA and its working fine at our end. the reader reads out indicating image/button can be clickable. `Microsoft Teams Version 1.4.00.32721`
Please try to update MS Teams client and check once if it helps

**Comment by prajwalmanju** [comment]
Hello Team,
The screen reader behavior is getting changed now a days, now it is reading as "Clickable button" but, it is not reading image associated alt text, and also even after read selected block as clickable button user not able to navigate to the specific block.

**Comment by Prasad-MSFT** [comment]
@prajwalmanju -We checked this , but NVDA reads out the associated alt text as well along with that it also indicates that the image is clickable. Please check again by updating and confirm.

**Comment by prajwalmanju** [comment]
Hello Team,

Any update on this issue ?

**Comment by v-chetsh** [comment]
@prajwalmanju - We have raised a bug for it and engineering team is working on it.
We will share you the updates but we do not have exact ETA to share.
